src/modulo1/tex/sections/limiti.tex

```diff
@@ -1049,16 +1049,16 @@ \subsection{Continuità di una funzione}
 La \textbf{continuità della tangente} nel suo dominio naturale è dovuta dal fatto che la tangente può essere definita come rapporto tra \textit{seno} e \textit{coseno}, e il rapporto di funzioni continue è anch'esso continuo. L'\textbf{esponenziale} è continuo. Tutte le \textbf{funzioni inverse delle funzioni elementari} sono continue ($\ln{x}$, $\sqrt{x}$, $\arcsin{x}$, $\arccos{x}$ e $\arctan{x}$).\\
 
 \subsubsection{Funzioni definite a tratti}
-Per le funzioni definite a tratti il caso è un po' più particolare perché non si sa a priori se sono continue o no. In generale il metodo per scoprire sono sono continue è il seguente. Prendiamo una funzione $f$ definita a tratti in questo modo:
+Per le funzioni definite a tratti il caso è un po' più particolare perché non si sa a priori se sono continue o no. In generale il metodo per scoprire se sono continue è il seguente. Prendiamo una funzione $f$ definita a tratti in questo modo:
 \begin{equation*}
     f(x) =
     \begin{cases*}
         g(x) \qquad (x \leq a)\\
         h(x) \qquad (x > a)
     \end{cases*}
 \end{equation*}
-Di solito sia $g$ che $h$ sono funzioni formate da composizioni di funzioni elementari, che rendono automaticamente continue le due funzioni. Il problema sorge quindi nel punto di separazione $a$. Per verificare che sia continua la funzionem $f$ bisogna fare in modo che sia $g$ sia $h$ si avvicinino ad $a$ con lo stesso valore, cioè che abbiano lo stesso limite per $x \to a$.
+Di solito sia $g$ che $h$ sono funzioni formate da composizioni di funzioni elementari, che rendono automaticamente continue le due funzioni. Il problema sorge quindi nel punto di separazione $a$. Per verificare che sia continua la funzione $f$ bisogna fare in modo che sia $g$ sia $h$ si avvicinino ad $a$ con lo stesso valore, cioè che abbiano lo stesso limite per $x \to a$.
 \begin{equation*}
     \lim_{x \to a^-} g(x) = \lim_{x \to a^+} h(x)
 \end{equation*}
-Inoltre è richiesto che il limite coincida con il valore della funzione $f(a)$. In questo caso non abbiamo fatto il test perché il valore della funzione era compreso nella funzione $g$ in quanto era definito per $\leq$ invece che un minore stretto. Se però questo non fosse il caso va inserito $f(a)$ nelle uguaglianze dei limiti scritti sopra.
+Inoltre è richiesto che il limite coincida con il valore della funzione $f(a)$. In questo caso non abbiamo fatto il test perché il valore della funzione era compreso nella funzione $g$ in quanto era definito per $\leq$ invece che un minore stretto. Se però questo non fosse il caso va inserito $f(a)$ nelle uguaglianze dei limiti scritti sopra.
```

Comment:
Mi sa che in fondo a questa linea c'è qualcosa che non va il carattere di a capo perché non vedo differenze di parole tra queste de righe